.github/workflows/testing-scheduled.yml

```
@@ -5,16 +5,14 @@ on:
     - cron: '30 5 * * 1' 
 
 
-jobs:     
-  test-stable-scheduled:
-    uses: 
-      ./.github/workflows/testing.yml
-    with:
-      xdist: no
-      branch_name: stable
-      event_name: ${{ github.event_name }}
-    secrets:
-      PAT: ${{ secrets.PAT }}
+jobs:
+  trigger_stable_test:
```

```suggestion
  trigger-testing-stable:
```

Just to keep a somewhat consistent naming scheme.